packages/calcite-components/src/components/action/action.scss

```diff
@@ -76,7 +76,9 @@
 
 :host([scale="s"]) {
   .button {
-    @apply text-n2h px-2 py-1 font-normal;
+    @apply text-n2h font-normal;
```

Comment:
Maybe these changes to action should be in a separate PR so this change can be tested and documented?

Reply:
Redesigned to avoid changes at `calcite-action` level.